scripts/post-processing/2-dim_reduction_analysis.R

```diff
@@ -0,0 +1,180 @@
+# C.Savonen, CCDL for ALSF
```

Comment:
Can we put a note _somewhere_ that these methods expect columns to be features? I think this could be helpful to people who are less familiar with these approaches if they're reading this

Reply:
Where would I put that note? Also, I guess I'm not exactly sure what you're trying to clarify.

Reply:
If someone is accustomed to working with gene expression data and not the methods, the transpose step might be surprising.

Reply:
I was thinking around line 93